trunk/NDHMS/Routing/module_channel_routing.F

```diff
@@ -784,9 +873,9 @@ Subroutine drive_CHANNEL(did, latval,lonval,KT, IXRT,JXRT, SUBRTSWCRT, &
                        tmpQLINK(k,2) = MUSKING(k,Qup,(Quc+QLateral(k)),QLINK(k,1),DTRT_CH,Km,X) !upstream plust lateral inflow
                    elseif (channel_option .eq. 2) then ! muskingum cunge
 
-                   call SUBMUSKINGCUNGE(tmpQLINK(k,2), velocity(k), LINKID(k),  &
+                   call SUBMUSKINGCUNGE(tmpQLINK(k,2), velocity(k), qloss(k), LINKID(k),  &
```

Comment:
Calls SUBMUSKINGCUNGE. I confirmed that the added qloss and ChannK variables are in the right place.